commit 1356b8c28d67cafd74f7e7dcfb39bf53681790a5
Author: Levent Serinol <lserinol@gmail.com>
Date:   Mon Mar 20 13:44:11 2006 -0500

    SUNRPC: more verbose output for rpc auth weak error
    
    This patch adds server ip address to be printed out when "server
    requires stronger authentication" error occured.
    
    Signed-off-by: Levent Serinol <lserinol@gmail.com>
    Signed-off-by: Trond Myklebust <Trond.Myklebust@netapp.com>

diff --git a/net/sunrpc/clnt.c b/net/sunrpc/clnt.c
index a6d7c332dd49..cad7efe2cb22 100644
--- a/net/sunrpc/clnt.c
+++ b/net/sunrpc/clnt.c
@@ -1197,8 +1197,8 @@ call_verify(struct rpc_task *task)
 			task->tk_action = call_bind;
 			goto out_retry;
 		case RPC_AUTH_TOOWEAK:
-			printk(KERN_NOTICE "call_verify: server requires stronger "
-			       "authentication.\n");
+			printk(KERN_NOTICE "call_verify: server %s requires stronger "
+			       "authentication.\n", task->tk_client->cl_server);
 			break;
 		default:
 			printk(KERN_WARNING "call_verify: unknown auth error: %x\n", n);